scripts/verify-handle-new-user-fix.mjs

```
@@ -0,0 +1,647 @@
+      setTimeout(() => {
+        if (pendingRequests.has(request.id)) {
+          pendingRequests.delete(request.id);
+          reject(new Error(`Request ${request.id} timeout`));
+        }
+      }, 30000);
+
+    } catch (error) {
+      reject(error);
+    }
+  });
+}
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)